src/bundle/Resources/config/fieldtype_services.yaml

```diff
@@ -145,3 +145,10 @@ services:
             - '%ibexa.field_type.richtext.converter.edit.xhtml5.resources%'
             - '@ibexa.config.resolver'
 
+    Ibexa\Bundle\FieldTypeRichText\Command\MigrateNamespacesCommand:
+        autowire: true
+        autoconfigure: true
+
+    Ibexa\FieldTypeRichText\Persistence\Legacy\ContentModelGateway:
+        autowire: true
+        autoconfigure: true
```

Comment:
Please ensure to inject proper database connection.

Reply:
Fixed in 8264f2d52fb01eaa02784f31a9e28f02e13c6f3e